src/clj/puppetlabs/services/jruby/jruby_puppet_service.clj

```diff
@@ -99,6 +99,12 @@
                                  (tk-services/service-context this))]
      (get-in @environment-class-info [env-name :info :classes])))
 
+  (get-environment-info-tag
```

Comment:
Did you leave the `class`-specific version of this function on purpose? Are we required to maintain the protocol or something? Because I think this commit gets rid of the only place it's called...

Reply:
So far this PR only changes names of those methods internal to jruby-puppet-core and then adds new functions where needed by external consumers. I'd love to remove all the old ones, replace them with generic ones, and remove "class" from any cache references, but that will add a lot of busy work that's orthogonal to the actual functionality and didn't know if I should do it here or in a follow on PR.

Reply:
Got it, this is what you were referring to in your top comment. Sure, it can wait for a follow-on.